c2corg_ui/static/partials/feedtable.html

```diff
@@ -0,0 +1,66 @@
+<div ng-show="wfeedCtrl.documents.length > 0"
+  infinite-scroll="wfeedCtrl.getDocumentsFromWhatsnewFeed()"
+  infinite-scroll-immediate-check="false"
+  infinite-scroll-distance="0.7"
+  infinite-scroll-disabled="wfeedCtrl.busy || wfeedCtrl.error || wfeedCtrl.feedEnd || wfeedCtrl.noFeed">
+
+  <div class="table-responsive">
+    <table class="text-center feedtable table table-condensed table-striped table-hover">
+
+      <p class="text-success text-center" ng-if="wfeedCtrl.userId"
+         ng-init="user = wfeedCtrl.userId;" ng-model="amount = wfeedCtrl.feedLength;">
+        This feed for the user {{user}} contains {{amount}} changes.
+      </p>
+      <p class="text-success text-center" ng-if="!wfeedCtrl.userId"
+         ng-model="amount = wfeedCtrl.feedLength;">
+        This feed contains {{amount}} changes.
+      </p>
+
+      <tr>
+        <th class="text-center" translate>title</th>
+        <th class="text-center" translate>type</th>
+        <th class="text-center" translate>written_at</th>
+        <th class="text-center" translate>quality</th>
+        <th class="text-center" translate>username</th>
+        <th class="text-center" translate>comment</th>
+      </tr>
+
+      <tr class="feed-list"
+              ng-repeat="doc in wfeedCtrl.documents track by doc.version_id"
+          ng-init="type = wfeedCtrl.documentType(doc.document.type); console.log(type);" >
+
+        <td class="text-left">
+          <a href="{{wfeedCtrl.getDocHref(doc)}}">{{::doc['document']['title']}}</a> ({{::doc['lang']}})
+          <span ng-if="doc['document']['version'] > 1 && (doc['document']['type'] !== 'i' && doc['document']['type'] !== 'o')">
+            (<a ng-href="{{wfeedCtrl.getVersionHref(doc)}}">previous version</a> | diff)
```

Comment:
Prev. version works.

I think that it is better to have those links (to document + to the previous version) separately. In a case of documents which have just one version, it is better to point to that document otherwise an user has no chance to find it.
It can be confusing to sometimes have a title's `href` pointing to the previous version and sometimes for latest document (if it was just created)...

Now when there are those document badges I have an idea if it is better to have link to the document on the badge and if the document has any previous version -> point to it with the title (and delete "previous version" text). What do you think?

![image](https://cloud.githubusercontent.com/assets/6165660/23060725/4669ff46-f4ff-11e6-92a0-e0a44c7c51c3.png)